content/en/docs/guides/mozilla-sops.md

```diff
@@ -182,7 +182,13 @@ You can now commit the encrypted secret to your Git repository.
 Note that you shouldn't apply the encrypted secrets onto the cluster with kubectl. SOPS encrypted secrets are designed to be consumed by kustomize-controller.
 {{% /alert %}}
 
-### Using various cloud providers
+## Encrypting secrets using age
+
+age is a simple, modern alternative to OpenPGP. It's recommended to use age over PGP, if possible.
+
+Encrypting with age follows the same workflow than PGP. See the [Kustomization-controller API doc](https://github.com/fluxcd/kustomize-controller/blob/main/docs/spec/v1beta2/kustomization.md#age) for more details.
```

Comment:
```suggestion
Encrypting with age follows the same workflow than PGP. See the [Kustomization-controller API doc](/docs/components/kustomize/kustomization/#age) for more details.
```

Should we maybe link to the same doc within the website instead?

Reply:
Does this works? what I see in `../components/kustomize/_index.md` is a link to the controller docs.

Reply:
Change it like Daniel suggest then check if it works in the preview website that netlify creates.

Reply:
great! 😊